src/apps/common/submodules/portWizard/portWizard.js

```diff
@@ -674,6 +838,40 @@ define(function(require) {
 			globalCallback();
 		},
 
+		/**************************************************
+		 *                  API requests                  *
+		 **************************************************/
+
+		/**
+		 * Lookups details for a list of numbers
+		 * @param  {Object} args
+		 * @param  {String[]} args.numbers Phone numbers to look up
+		 * @param  {Function} args.success  Success callback
+		 * @param  {Function} [args.error]  Error callback
+		 */
+		portWizardRequestPhoneNumbersLookup: function(args) {
+			var self = this;
+
+			monster.request({
+				resource: 'phonebook.lookupNumbers',
+				data: {
+					data: {
+						numbers: args.numbers
+					}
+				},
+				success: function(data) {
+					args.success(data.data);
+				},
+				error: function(data, status) {
+					var errorData = {
+						phonebookUnavailable: data.status === 0
+					};
+
+					_.has(args, 'error') && args.error(errorData);
+				}
```

Comment:
When you bypass the global error handler, you should always call it once your custom error handling is done to catch unexpected errors.

In this case, you might also want to add `500` to the list of status codes to handle.